scripts/merge_master_to_branches.py

```diff
@@ -0,0 +1,107 @@
+import os
+import subprocess
+
+def get_release_branches(prefix="new-feature/mcus"):
```

Comment:
@IvanRuzavin As this is prone to fail when run from GITHUB, comment out the needed code and just leave the merge to one test branch. Once test is confirmed, we will move on from there.

Create a test branch not from main, but one the previous stale branches.